src/libstore/misc.cc

```diff
@@ -83,14 +83,16 @@ void Store::computeFSClosure(const StorePath & startPath,
 }
 
 
-std::optional<ContentAddress> getDerivationCA(const BasicDerivation & drv)
+const ContentAddress * getDerivationCA(const BasicDerivation & drv)
```

Comment:
```suggestion
const ContentAddress * getFixedOutputAddress(const BasicDerivation & drv)
```

 - "CA" doesn't convey anything not already in the type name.
 - I think "derivation" is ok to be implied.

Reply:
The method name predated this PR, so I am a bit wary of changing it.

However, I did change its type, so maybe I am just being silly.

Reply:
Just a suggestion. Resolving.